db/migrate/20160114120200_rename_job_categories.rb

```diff
@@ -0,0 +1,20 @@
+class RenameJobCategories < ActiveRecord::Migration
+  MAP = {
+    'Package Action - SSH Default' => ['Package Action', 'Packages'],
```

Comment:
The conclusion was not to rename the template name and keep the `SSH Default` there, right?